client/configuration.js

```diff
@@ -0,0 +1,4 @@
+const configuration = {
```

Comment:
This file was not versioned to avoid configuration problems in the different environments